### ISSUE TITLE
Space stations disappearing

### ISSUE THREAD

**Issue by stephen92** [zmaster587/AdvancedRocketry#1237]
## Version of Advanced Rocketry
AdvancedRocketry 1.12.2-1.4.0-68-UNSTABLE build -68 for 1.12.2
## Have you verified this is an issue in the latest unstable build
n/a
## Version of LibVulpes
LibVulpes-1.12.2-0.2.8-27-universal.jar
## Version of Minecraft
1.12.2
## Description of the problem
Using Sponge 1.12.2 this is what I have done to try and fix the issue:
sponge/worlds/advancedrocketry/space/dimension.conf
        # If 'true', this worlds spawn will remain loaded with no players.
        keep-spawn-loaded=true

Do I need to enable load-on-startup as well?

you create a space station and send it to space - it works fine and dandy. You spend many hours creating a wonderful intergalactic base. You are happy and proud and go to bed for the night. You then come on the next morning and found your space station has vanished mysteriously. You even had a home set there but when you try to teleport to that you get a message saying something such as: "You cannot teleport here as it is not a valid point."

**Comment by GmanAtheist** [comment]
Same problem here. My space station was there last night. Now it is gone. More like it is there but I can not land on it. Even my portal does not work. Space station comes past me every 10 or so seconds and then disappears in the distance. Why isn't the mod developer respond , or anyone who has had this problem?.

**Comment by dmodoomsirius** [comment]
You are running an old build. Update to build 88 and see if it happens still.

**Comment by GmanAtheist** [comment]
Built a massive lovely space station. Worked without problems. Then came back into game next day and I can not land on my space station. The station comes and goes fast past me while I am just floating in space. I loaded a older version (AdvancedRocketry-1.7.10-1.0.19b) , (LibVulpes-1.7.10-0.1.19-universal) and space station is back stable. But without space elevator and beacons are not on the overworld.

I have now installed AdvancedRocketry-1.7.10-1.2.5 and LibVulpes-1.7.10-0.2.6-universal. Space station is back and all looks OK for now.  I will report any issues I may have so as to help others. :-)
